src/angular/components/send/add-edit.component.ts

```diff
@@ -1,11 +1,6 @@
 import { DatePipe } from '@angular/common';
 
-import {
-    EventEmitter,
-    Input,
-    OnInit,
-    Output,
-} from '@angular/core';
+import { Directive, EventEmitter, Input, OnInit, Output } from '@angular/core';
```

Comment:
I believe the pattern for imports here would be after 2 items, to put them on separate lines as it was.